src/components/home/modals/Send.vue

```diff
@@ -64,6 +64,8 @@
             <b-form-invalid-feedback id="inputLiveFeedback">
               Invalid recipient address (if using QR code scanner, make sure QR code is correct).
             </b-form-invalid-feedback>
+            <p v-if="superNodes.indexOf(recipient) > -1"
+               class="super-node">Caution! It's super-node address. </p>
```

Comment:
Change the words to "Attention: You are doing PAYMENT, not LEASE, to super node address. You cannot get coins back once sent. Please confirm your action!"